traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -107,6 +111,8 @@ sub update {
 	}	
 
```

Comment:
right under line 95, can you just do a check to see if they are allowed to edit this tenant?

if (!verify_tenancy_for_write($self, $params->{id})) {
return $self->alert("This tenant is not under user's tenancy.");
}